WcaOnRails/app/models/registration.rb

```diff
@@ -171,6 +171,12 @@ def waiting_list_info
     OpenStruct.new(index: index, length: pending_registrations.length)
   end
 
+  def paid_waiting_list_info
+    pending_paid_registrations = competition.registrations.pending.order(registration_payments.map(&:created_at).max)
+    index = pending_paid_registrations.index(self)
+    OpenStruct.new(index: index, length: pending_paid_registrations.length)
```

Comment:
Avoid using `OpenStruct`, it is deprecated. With the newer RuboCop versions, the overcommit checks will also warn about this.

Reply:
I understand that you did this only because the above `waiting_list_info` does the same. But if you pull in the latest changes, you will notice that I have already changed that :D